### PR TITLE
Fixes

### LINKED ISSUE #210: Minor adjustments
- Minimum filter size should be 0.5, not 0.0.
- Exported projects have Max Diffuse Bounce set to 0 for the interactive configuration, that doesn't match what's set in blenderseed.

### PR DESCRIPTION
**Pull request by jdent02**
#210

### CODE REVIEW

**Comment by jdent02** [comment]
#212

File: docs/panels/render/lighting.rst [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+.. image:: /_static/screenshots/render_panels/lighting.JPG
+
+|
+
+- Next Event Estimation:
+	- Whether or not light sources are directly sampled in addition to being hit by indirect rays.
+- Direct Lighting:
+	- Whether or not discreet light sources (point, spotlight and sun) are directly sampled at each surface hit point.
+- Low Light Threshold:
+	- This prevents shadow rays from being traced to lights that do not contribute significant illumination to the surface hit point.  Higher numbers can lead to a decrease in lighting quality but an increase in convergence.
```
Comment (dictoon) [inline docs/panels/render/lighting.rst]: *an increase in convergence **speed*** perhaps?

File: docs/panels/render/lighting.rst [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+-------------------
+
+.. image:: /_static/screenshots/render_panels/lighting.JPG
+
+|
+
+- Next Event Estimation:
+	- Whether or not light sources are directly sampled in addition to being hit by indirect rays.
+- Direct Lighting:
+	- Whether or not discreet light sources (point, spotlight and sun) are directly sampled at each surface hit point.
```
Comment (dictoon) [inline docs/panels/render/lighting.rst]: *Sun* is capitalized in English, I think.

Comment (dictoon) [inline docs/panels/render/lighting.rst]: I love the term "discreet lights", that's how they should have been called in the code! Might do a big rename pass at some point...

File: docs/panels/render/lighting.rst [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+	- Whether or not an environment map can contribute lighting to the scene.
+- Caustics:
+	- Whether or not caustic effects are rendered.
+- Bounces:
+	- Global: Sets a maximum number of bounces for a light ray.
+	- Diffuse: Sets a maximum number of diffuse bounces a ray can make.
+	- Glossy: Sets the number of glossy bounces a ray can make.
+	- Specular: Sets the number of specular bounces a ray can make.
+	- Volume: Sets the number of bounces a ray can make inside a volume.
+	- Max Ray Intensity: Sets the maximum brightness a ray can contribute to the scene.  Lowering this can remove fireflies at the expense of accurate lighting.
```
Comment (dictoon) [inline docs/panels/render/lighting.rst]: Note that MRI only applies to secondary rays.

File: docs/panels/render/lighting.rst [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+- Russian Roulette Start Bounce:
+	- Sets the bounce after which Russian Roulette will be used to terminate rays that are not expected to contribute much to the final image.  Lowering this can increase noise but speed up rendering.
```
Comment (dictoon) [inline docs/panels/render/lighting.rst]: Great explanation!

File: docs/panels/render/render.rst [hunk 5]
```diff
@@ -5,11 +5,13 @@ Render
...
+- Render Frame:
+	- Exports and renders the current Blender frame.
+- Render Animation:
+	- Exports and renders all frames in between the start and end points defined in the dimensions panel.
+- Export Frame:
+	- Exports the scene file for the current frame.
+- Export Animation:
+	- Exports the scene files for each frame in between the start and end points defined in the dimensions panel.
+- Display:
+	- This defines how the in progress render is displayed within Blender.
```
Comment (dictoon) [inline docs/panels/render/render.rst]: *in-progress* maybe?

File: util.py [hunk 6]
```diff
@@ -52,17 +52,17 @@ def get_osl_search_paths():
...
+    osl_info_dir = os.path.join(appleseed_parent_dir, 'bin')
+
+    return osl_info_dir, shader_directories
 
 
 def read_osl_shaders():
     """Compiles any .osl files and reads all .oso parameters"""
 
     if bpy.context.user_preferences.addons['blenderseed'].preferences.appleseed_binary_directory == "":
         print("appleseed binary path not set.  Rendering and OSL features will not be available")
```
Comment (dictoon) [inline util.py]: What happens after this message is printed? Shouldn't we exit `read_osl_shaders()` altogether?

File: util.py [hunk 7]
```diff
@@ -52,17 +52,17 @@ def get_osl_search_paths():
...
-        return
-
-    appleseed_bin_dir, shader_directories = get_osl_search_paths()
+    osl_info_dir, shader_directories = get_osl_search_paths()
```
Comment (dictoon) [inline util.py]: Add blank line above this to separate it from the rest.

Comment (jdent02) [reply util.py]: Hmm, I thought there was a 'return' after that line.  I'll add it in.

**Comment by jdent02** [comment]
This should be okay to merge now.  The AOV stuff has been removed.

File: docs/panels/world/sss_sets.rst [hunk 8]
```diff
@@ -1,2 +1,21 @@
 SSS Sets
-========
\ No newline at end of file
+========
+
+.. image:: /_static/screenshots/world_panel/sss_sets.JPG
+
+|
+
+SSS sets are a way of grouping several objects together into a common 'container' for SSS shading.  Grouping objects together in this way can avoid shading artifacts.
```
Comment (dictoon) [inline docs/panels/world/sss_sets.rst]: SSS sets are really a way to consider a group of object to be the same object, allowing light to flow from one object to another.

**Comment by dictoon** [comment]
Looking good! Could you merge related commits together please? Probably one for the docs and one for the oslinfo fix?

**Pull request merged**